README.md

```diff
@@ -47,13 +47,13 @@ For more info refer to,
 [documentation]: http://autumnai.github.io/leaf
 
 > Disclaimer: Leaf is currently in an early stage of development.
-> If you are experiencing any bugs that are not due to not yet implemented
-> features, feel free to create a issue.
+> If you are experiencing any bugs with features that have been
+> implemented, feel free to create a issue.
```

Comment:
It was actually meant to say the opposite, but I can't really remember why, and I think your change works just as well.